go.mod

```diff
@@ -22,21 +22,21 @@ require (
 	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.44.1
 	github.com/prometheus-operator/prometheus-operator/pkg/client v0.0.0-00010101000000-000000000000
 	github.com/prometheus/alertmanager v0.21.0
-	github.com/prometheus/client_golang v1.8.0
-	github.com/prometheus/common v0.14.0
+	github.com/prometheus/client_golang v1.9.0
+	github.com/prometheus/common v0.15.0
 	github.com/prometheus/prometheus v1.8.2-0.20201015110737-0a7fdd3b7696
```

Comment:
do we want to bump prometheus and thanos versions here too?

Reply:
Can do!

Reply:
Done, any other ones. I didn't go through the minor ones as we have a lot, but if you can think of any other ones happy to bump!